src/Applications/LDAS_App/util/postproc/climatology/Write_L4_sm_clim_stat_bin2nc4.m

```diff
@@ -0,0 +1,208 @@
+%=========================================================================
+% script to convert SMAPL4 SM climatology statistic binary output to nc4
+% file. 
+%
+% The tile_ids are in the same order as in ldas_tilecoord.txt 
+%
+% Q. Liu       - Jun 20, 2016
+% G. De Lannoy - Apr 21, 2015
+%=========================================================================
+
+clear all
+
+write_GEOSldas_tileorder = 0;
+
+time_stamp = '2001_p1_2021_p73';
+specs_tag  = '_W_5p_Nmin_210';
+
+bin_fpath  = '/home/qliu/smap/SMAP_Nature/SMAP_Nature_v10/SMAP_Nature_v10.0/output/SMAP_EASEv2_M09_GLOBAL/stats/cli/';
+
+file_out   = ['L4SM_NRv10.0_cli_',time_stamp,specs_tag,'_sm_wetness_EASEv2_M09.nc4'];
```

Comment:
@gmao-qliu: Aren't the percentile values in soil moisture units (m3/m3) rather than dimensionless wetness units?  The gph output variables are sm_rootzone_pctl and sm_profile_pctl.  Using "wetness" in the file name, nc4 attributes, and labels seems confusing.  Maybe this can be cleaned up?  Or am I misunderstanding something?

Reply:
@gmao-qliu: After further inspection, I think the percentile values that are generated by `get_model_clim_stats.m` are indeed in dimensionless wetness units.  I'm not sure why.  Presumably, the L4 ops python code that converts soil moisture into pctl units must make a matching unit conversion.  
In any case, we should track down the L4 ops python code that uses the clim file and also add it to this repository. 
Ideally, we would also make the computations in `get_model_clim_stats.m` independent of the L4SM-specific post-processing.  The `get_model_clim_stats.m` should really be a generic utility that takes a given field from any GEOSldas output collection and generates the stats and percentile values, without unit conversions.
Maybe I'm still misunderstanding what's going on, but I'm getting a sense that the scripts in their present form could be improved.

Reply:
@gmao-rreichle gph contains wetness output. The python script takes those to compute percentiles. The units conversion in get_model_clim_stat.m is necessary because the Nature Run tavg3h output doesn't include wetness. The script can be modified and used for other variables, but I'm hesitating to make the change now without testing on individual variables with broader value ranges.

Reply:
@gmao-qliu, thanks for the insights.  I had thought that the L4 ops script uses vol soil moisture as input but couldn't check because of the Discover outage yesterday.  In any case, I cleaned up some more and resurrected the "out_wetness" switch to deal with the unit conversion.  It's minimally invasive and I'm reasonably confident that this will work ok.  Please take a look: https://github.com/GEOS-ESM/GEOSldas/pull/569/commits/3364e14062ffd23a3c8eb71b670b3527c8d80803?diff=split&w=1  (white-space changes hidden).

Reply:
@gmao-rreichle I just verified the changes. It works well.